src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import classNames from 'classnames';
+import _ from 'lodash';
+import { bindClassNames } from '../../util/style-helpers';
+
+const boundClassNames = bindClassNames('Banner');
+
+const {
+	bool,
+	string,
+	oneOfType,
+	oneOf,
+	node,
+	arrayOf,
+	func,
+	element
+} = React.PropTypes;
+
+const defaultIcons = {
+	'success': '<SuccessIcon/>',
+	'danger': '<DangerIcon/>',
+	'info': '<InfoIcon/>',
+	'warning': '<WarningIcon/>',
+	'primary': null,
+	'default': null
+};
+
+/**
+ *
+ * {"categories": ["controls", "Banners"]}
```

Comment:
Categories should be lower case. The code that reads them passes them through the [`_.startCase`](https://lodash.com/docs#startCase) function to format them.